apps/web/src/components/Footer.tsx

```diff
@@ -1,20 +1,76 @@
 import React from 'react';
-import { useNavigate } from 'react-router-dom';
+import { ReactComponent as CallIcon } from 'src/images/footer_call.svg';
```

Comment:
Why do we need this? can we just do like below?
`
import call from 'src/images/footer_call.svg';
<img src={call} />
`

Reply:
Under the circumstances, this is a good approach. This is being used as an icon rather than an image. Also, img tag was making the render have some antialiasing I was not liking.